### PR TITLE
tests: Propagate additional CA when using operator too.

### PR DESCRIPTION
**Pull request by porridge**
### Description

This lets us run `TestCASetup` on OCP too, see one of the issues in ROX-24688.

### User-facing documentation

(*must be* 2 items and both *must be* checked)

- [x] CHANGELOG update is not needed
- [x] Documentation is not needed

### Testing

- [x] inspected CI results

#### Automated testing

(*must be* at least 1 item and all items *must be* checked)

- [x] modified existing tests
  

#### How I validated my change

CI should be enough.

### CODE REVIEW

File: tests/e2e/lib.sh [hunk 1]
```diff
@@ -237,6 +237,8 @@ deploy_central_via_operator() {
     ROX_PASSWORD="$(tr -dc _A-Z-a-z-0-9 < /dev/urandom | head -c12 || true)"
     centralAdminPasswordBase64="$(echo "$ROX_PASSWORD" | base64)"
 
+    centralAdditionalCA="$(sed 's,^,        ,' "${TRUSTED_CA_FILE}")"
```
Comment (janisz) [inline tests/e2e/lib.sh:240]: Why do we need to indent every line? Can you add a comment with explanation or add it in PR description?

Comment (porridge) [reply tests/e2e/lib.sh]: Renamed the variable to make it self-explanatory.

**Comment by porridge** [comment]
This needed a couple more fixes, PTAL at the new commits @janisz 
Sorry for the noise 🥺

File: tests/e2e/yaml/central-cr-midstream.envsubst.yaml [hunk 2]
```diff
@@ -34,6 +34,11 @@ spec:
     persistence:
       persistentVolumeClaim:
         claimName: stackrox-db
+${disableSpecTLS}  tls:
+${disableSpecTLS}    additionalCAs:
+${disableSpecTLS}    - name: additional-ca
+${disableSpecTLS}      content: |
+$centralAdditionalCAIndented
```
Comment (janisz) [inline tests/e2e/yaml/central-cr-midstream.envsubst.yaml:41]: Don't we need disabled TLS here?

**Review by janisz — APPROVED** [review]

**Pull request merged**